slacker/__init__.py

```diff
@@ -187,8 +187,8 @@ def admin(self):
     def info(self, user):
         return self.get('users.info', params={'user': user})
 
-    def list(self, presence=False):
-        return self.get('users.list', params={'presence': int(presence)})
+    def list(self, cursor=None, limit=None, presence=False):
```

Comment:
Can you add these two optional parameters at the end? I just want to make sure that we won't break when it's called with positional arguments.